docs/examples/webhook.js

```diff
@@ -6,8 +6,13 @@
 
 // Import the discord.js module
 const Discord = require('discord.js');
-
-// Create a new webhook
+/*
+ * Create a new webhook
+ * Your webhook's id and token can be found at the URL or in the response body
```

Comment:
```suggestion
 * The webhook's id and token can be found in the URL or in the response body
```
i think this would bebetter wording, also should probably say what 'response body' is being referred to here

Reply:
Yes, I agree thank you Sugden.